### PR TITLE
Provide link to old file viewer for browse view

### LINKED ISSUE #1471: Provide link to old file viewer for browse view
In https://github.com/mozilla/addons-server/issues/13634 we're discussing replacing all links to the old file viewer in AMO by links to the code manager. 

We need to add a way to fall back to the old file viewer if things go wrong in code-manager. Later when we sunset the old file viewer we'll then remove that link.

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #1471 

Before:

Screenshot 2020-03-05 12 38 05

After:

Screenshot 2020-03-05 12 37 06

### CODE REVIEW

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -199,6 +203,16 @@ export class NavbarBase extends React.Component<Props, State> {
           </div>
         </Navbar.Brand>
         <Navbar.Text className={styles.text}>
+          {currentVersion && file ? (
+            <a
+              className={styles.legacyLink}
+              href={`${reviewersHost}/${process.env.REACT_APP_DEFAULT_API_LANG}/firefox/files/browse/${file.id}/file/${currentVersion.selectedPath}`}
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/components/Navbar/index.tsx]: I was assuming that we wanted to direct the user to the correct file in the legacy file viewer. If we only want to send them to the root of the file viewer that would be easier, but it seems less user-friendly. If we do want to re-direct to the file, the values I have used should be correct. I did a bunch of local testing, and the `lang` seems to be needed.

@wagnerand Do you have an opinion about whether we should link to the same file in the legacy viewer, or just the root?

File: src/components/Navbar/index.tsx [hunk 2]
```diff
@@ -199,6 +203,16 @@ export class NavbarBase extends React.Component<Props, State> {
...
+              rel="noopener noreferrer"
+              target="_blank"
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/components/Navbar/index.tsx]: Yes, it's probably okay, but I was thinking that, as our intention is to encourage them to use the new viewer as much as possible, keeping that open in a separate tab might be preferable to redirecting them to the legacy viewer at which point they might be more inclined to stay there.

File: src/components/Navbar/index.tsx [hunk 3]
```diff
@@ -31,10 +33,11 @@ export type DefaultProps = {
 
 type PropsFromState = {
   comments: Comment[] | undefined;
-  user: User | null;
   currentBaseVersion: Version | null | undefined | false;
   currentBaseVersionId: number | undefined | false;
   currentVersion: Version | null | undefined | false;
+  file: VersionFile | null | undefined;
+  user: User | null;
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/components/Navbar/index.tsx]: I didn't add `user`, I simply moved it down in this block of code to make them alphabetical.

**Comment by bobsilverberg** [comment]
As I am going to disappear for two weeks after tomorrow, I think I'll merge this and allow @wagnerand to respond to https://github.com/mozilla/addons-code-manager/pull/1473#discussion_r388490786 later. If a follow-up needs to be filed at that time we can do that.

File: src/components/Navbar/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (wagnerand) [reply src/components/Navbar/index.tsx]: Linking to same file is fine.

**Pull request merged**